### ISSUE TITLE
Missing form label on Search field

### ISSUE THREAD

**Issue by tomsedory** [OpenEugene/little-help-book-web#176]
The Search field is missing a label. See this screen shot from the [WAVE extension](https://wave.webaim.org/extension/) that evaluates http sources for [WCAG compliance](https://www.w3.org/WAI/standards-guidelines/wcag/).

![image](https://user-images.githubusercontent.com/75692630/104685584-406c3c00-56b0-11eb-9781-1567a74075d0.png)

Note: Missing form labels is one of the top accessibility issues for users using screen readers.
